### ISSUE TITLE
How do I compare branches?

### ISSUE THREAD

**Issue by bontchev** [eamodio/vscode-gitlens#604]
I would like to be able to compare different git branches from within VSCode. Googling around lead me to this extension. The documentation (which is really a humongous list of features) suggests that it indeed can compare branches - however, I am unable to figure out how exactly to do it.

There seems to be (according to the documentation) something called "Compare view" - but I have no idea how to display it, either.

Intuitively, I would expect to open the Gitlens group in Explorer, then right-click on a branch in the Branches subtree and select to compare it with the current branch from the pop-up menu. However, the pop-up menu does not offer such a function:

![clipboard01](https://user-images.githubusercontent.com/580507/50424719-aab7a700-0871-11e9-9772-55c9c09ab0f5.jpg)

So, could somebody please explain me how to do it?

**Comment by bontchev** [comment]
Thank you. My version of GitLens is 6.4.0 - that's what VSCode installed from the repository. It is possible, however, that I don't have the latest version of VSCode. I'll see if I can install a newer version and it it will update GitLens to a newer version.

**Comment by bontchev** [comment]
Yep, that was it. Once I managed to install VSCode 1.30.1 (a somewhat non-trivial task, because the installer kept hanging when I told it to install over the existing installation in `C:\Program Files\VScode`), GitLens updated to 9.2.4 and everything looks very different (there is no Gitlens group in Explorer; instead there is a Gitlens icon on the sidebar) but, most importantly, I can now compare branches.

Thanks again for your help. Closing the issue now, since my question was answered.
